packages/seedless-onboarding-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/seedless-onboarding-controller",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Considering the presence of breaking changes, should we communicate this by bumping the major part of the version instead of the minor part?

``` suggestion
  "version": "2.0.0",